assets/js/googlesitekit/data/create-settings-store.js

```diff
@@ -169,12 +169,13 @@ export const createSettingsStore = ( type, identifier, datapoint, {
 		 */
 		*saveSettings() {
 			const registry = yield Data.commonActions.getRegistry();
-			const values = registry.select( STORE_NAME ).getSettings();
+			registry.dispatch( STORE_NAME ).clearError( 'saveSettings', [] );
```

Comment:
See above comment, we should only call this if the store has it.

Reply:
Updated.